corehq/apps/userreports/reports/builder/columns.py

```diff
@@ -236,7 +234,7 @@ def _get_datatype(self, aggregation):
         return map[self._get_aggregation_config(aggregation)]
 
     def get_indicator(self, aggregation, is_multiselect_chart_report=False):
-        column_id = get_column_name(self._property) + "_" + self._get_datatype(aggregation)
+        column_id = get_column_name(self._property, suffix=self._get_datatype(aggregation))
```

Comment:
This is a pretty terrifying change where the failure mode is going to be a massive amount of table rebuilds due to differing column names. How confident are you that this will have exactly the same output as before? It may be worthwhile locally testing enikshay and ICDS reports to ensure that they are exactly the same before and after

Reply:
I think this would only affect report builder reports, not all reports.

Though generally agree that we should ideally not the existing column IDs that are already short enough (Ethan, does this change all column IDs?)

Reply:
The three of us offlined about this briefly.  The test file in this PR has some tests of this function which pass before and after this change.  My expectations are:
* The behavior of `get_column_name` will not be different for usages where `suffix` is not provided
* Where `suffix` _is_ provided, the behavior should be the same for columns that were short enough
* Places where `suffix` is provided and the column names with datatypes appended are too long are already broken.  They are truncated by [`truncate_value`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/util.py#L164-L164).  This PR will make the names match what's being saved in postgres (so it WILL change the names in the data sources once the report is saved).